app/templates/main/userProfile.html

```diff
@@ -365,39 +365,29 @@ <h6>{{bgType.description}}</h6>
               {% else %}
                 {% set anyAllowed = namespace(value=False) %}
                 {% for bgType in backgroundTypes %}
+                  {% set allowed = namespace(items=[]) %}
```

Comment:
allowed is also a confusing variable. make sure to go over all of them and make them more clear